projects/arlas-components/src/lib/components/mapgl/mapgl.component.html

```diff
@@ -33,9 +33,8 @@
   </div>
 </div>
 
-<div class="current-coordinate" *ngIf="displayCurrentCoordinates">
-  <span class="lat"> Lat : {{currentLat | number:'1.5-5'}} </span>
-  <span class="lng"> Lng : {{currentLng | number:'1.5-5'}} </span>
+<div *ngIf="displayCurrentCoordinates">
```

Comment:
Could be ng-container to limit the amount of elements in the DOM

Reply:
fixed